web/marketing/pages/stop-free-credit.html

```diff
@@ -0,0 +1,30 @@
+<!-- Copyright (C) 2019 Storj Labs, Inc.
+See LICENSE for copying information. -->
+
+{{define "stopFreeCredit"}}
+<div class="modal fade stop-free-credit-modal mt-5" tabindex="-1" role="dialog" aria-labelledby="confirm" aria-hidden="true">
+    <div class="modal-dialog modal-lg">
+        <div class="modal-content">
+            <div class="row d-flex justify-content-between">
+                <img class="mt-3 ml-5" src="/static/img/warning.png" height="30" width="30"/>
+                <p class="mt-3">Replace 
+                    <strong>{{.FreeCredits.Active.Name}}</strong> with 
+                    <strong>{{.FreeCredits.Default.Name}}</strong> ?
+                </p>
+                <div class="mt-2 mb-2 d-flex">
+                    <a href="/">
+                        <button class="btn btn-outline-secondary mr-1">
+                            <strong>X</strong>
```

Comment:
Use proper text `Cancel`, it gives better indication for what's going to happen.